### DISCUSSION TITLE
Custom Dialog

### DISCUSSION THREAD

**Discussion by Yara96** [rexrainbow/phaser3-rex-notes#200] [Q&A]
Hello,

Can you elaborate on how I can create a custom dialog? I just want to add a text box where user can enter a number, and a increment and decrement buttons next to it, in addition to having content and description labels. 

Thanks!

**Comment by rexrainbow** [comment]
Use sizer to create this custom dialog
- [Example](https://github.com/rexrainbow/phaser3-rex-notes/blob/master/examples/ui-sizer/number-input-dialog.js)
- [Live demo](https://codepen.io/rexrainbow/pen/jOwWmMm)

**Reply by Yara96** [reply]
Great, thank you!
